### PR TITLE
Restrict Number of Comments 

### PR DESCRIPTION
**Pull request by gracetian6**
Allows users to restrict the number of comments by editing a form input. This dynamically updates the web page once the number of comments gets updated and stores the number of comments when it gets refreshed. Demo here: https://drive.google.com/file/d/1wN6DkxbB4IyM5LCZKZzbm0VZWG_YbMuU/view?usp=sharing

### CODE REVIEW

File: portfolio/src/main/webapp/script.js
Comment (jessicaslaughter) [inline portfolio/src/main/webapp/script.js]: nit: remove space before semicolon

File: portfolio/src/main/webapp/script.js
Comment (jessicaslaughter) [inline portfolio/src/main/webapp/script.js]: nit: use single quotes around numComments

**Review by jessicaslaughter — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/script.js
Comment (jpnickolas) [inline portfolio/src/main/webapp/script.js]: why is this named `knit`? You should name it something like `init` or `onLoad`.

File: portfolio/src/main/webapp/script.js
Comment (jpnickolas) [inline portfolio/src/main/webapp/script.js]: nit: this should be named getComments (plural) since you'll be grabbing multiple comments.

File: portfolio/src/main/webapp/script.js
Comment (jpnickolas) [inline portfolio/src/main/webapp/script.js]: nit: this should be getMaxComments (plural)

File: portfolio/src/main/webapp/script.js
Comment (jpnickolas) [inline portfolio/src/main/webapp/script.js]: nit: you should set a default value here for when a user first visits your page:
```
const numComments = url.searchParams.get("numComments") || 5;
```

**Review by jpnickolas — CHANGES_REQUESTED** [review]

**Comment by jpnickolas** [comment]
Please wait for both Jess and I to review and approve these changes before merging htem in.

**Pull request merged**
